### PR TITLE
Feature/Model Dyslipidemia NCD

### PR DESCRIPTION
**Pull request by nobert-mumo**
Jira ticket: https://thepalladiumgroup.atlassian.net/browse/KHP3-4638

- Adding a new NCD - Dyslipidemia in the fact
- Availing a boolean column in the reporting layer for Dyslipidemia

### CODE REVIEW

File: Scripts/NDWH/C&T FACT TABLES/load_FactNCDs.sql [hunk 1]
```diff
@@ -108,6 +109,18 @@ diabetes_ordering as (
...
+    select 
+        PatientPKHash,
+        PatientPK,               
+        SiteCode,
+        VisitDate,
+        ChronicIllness,
+        row_number() over (partition by PatientPK, Sitecode order by VisitDate asc) as rank
+    from ODS.dbo.CT_AllergiesChronicIllness as chronic
+    where chronic.voided = 0 
+        and ChronicIllness like '%Dyslipidemia %'
```
Comment (cbrianbet) [inline Scripts/NDWH/C&T FACT TABLES/load_FactNCDs.sql:122]: @nobert-mumo  will the space at the end of  `Dyslipidemia` affect the results of the query?

Comment (nobert-mumo) [reply Scripts/NDWH/C&T FACT TABLES/load_FactNCDs.sql:122]: nice catch @cbrianbet. This indeed affects. I will update. Thanks

Comment (nobert-mumo) [reply Scripts/NDWH/C&T FACT TABLES/load_FactNCDs.sql]: @cbrianbet I have updated.

**Review by cbrianbet — APPROVED** [review]
LGTM!

**Review by Marymary-dev — APPROVED** [review]
Looks good to me

**Pull request merged**